docs-src/real_time_messaging.rst

```diff
@@ -47,6 +47,33 @@ If there was a problem connecting an error will be returned, including a descrip
     }
   }
 
+rtm.start vs rtm.connect
+---------------------------
+
+If you expect your app to be used on large teams, we recommend using starting the RTM client with `rtm.connect` rather than `rtm.start`, which is the default connection method for this client.
+`rtm.connect` provides a lighter initial connection payload, without the team's channel and user information included. You'll need to request channel and user info via
+the Web API separately.
+
+To do this, simply pass `with_team_state=False` into the `rtm_connect` call, like so:
```

Comment:
the word "this" is ambiguous and might refer to "requesting channel and user info via the Web API". maybe say "To get a lighter connection payload, ..."